conf/development.ini

```diff
@@ -22,15 +22,6 @@ secret_key: notverysecretafterall
 
 sqlalchemy.url: postgresql://postgres@localhost/postgres
 
-[server:main]
-use: egg:gunicorn#main
-host: 0.0.0.0
-port: 5000
-proc_name: web
-graceful_timeout: 0
-timeout: 0
-errorlog: -
```

Comment:
Renamed `development-app.ini` to just `development.ini`, the same as our other apps.

This `[server:main]` sections contained Gunicorn config settings. Gunicorn only reads settings from the `[server:main]` block of the PasteDeploy config file if Gunicorn is being run indirectly via `pserve`. We're no longer using `pserve`, so these Gunicorn settings have now moved into `gunicorn-dev.conf.py`.